lerna.json

```diff
@@ -2,5 +2,10 @@
   "packages": [
     "packages/**"
   ],
+  "command": {
+    "version": {
+      "message": "[version bump]: publish %s\nlerna.json\nCHANGELOG.md\n**/CHANGELOG.md"
```

Comment:
What does this line do?

Reply:
It changes the commit message.

`[version bump]` is needed for the deployment pipeline to recognize this is a version bump, and file matching patterns are required on the next lines to avoid needing QA approval when those files are changed.

Reply:
Could you explain the current behaviour and what is expected to happen after this change?

Also, why does the message have `%s` and `\n`, and `*` characters? Could the formatting be removed?
e.g. `publish lerna.json CHANGELOG.md`

Reply:
Currently, we can't deploy alpha releases without manual QA approval since version bump commits are being interpreted as actual changes. After this change, we should be able to deploy alpha releases without QA approval (or at-least that's my understanding).

As for the characters:
* `%s` is supposed to be replaced by the version string, according to the [lerna documentation](https://github.com/lerna/lerna/tree/main/commands/version#user-content---message-msg:~:text=lerna%20version%20%2Dm%20%22chore(release)%3A%20publish%20%25s%22).
* `[version bump]`, `\n` and `**` are meant to follow the [guidelines of the deployment pipeline](https://github.com/coveo/deployment-pipeline/blob/master/deployment_package/certifiers/package_rollout/audit_scm_changeset.py#L88).

Reply:
Currently, we can't deploy without manual QA approval since version bump commits are being interpreted as actual changes. After this change, we should be able to deploy without QA approval (or at-least that's my understanding).

As for the characters:
* `%s` is supposed to be replaced by the version string, according to the [lerna documentation](https://github.com/lerna/lerna/tree/main/commands/version#user-content---message-msg:~:text=lerna%20version%20%2Dm%20%22chore(release)%3A%20publish%20%25s%22).
* `[version bump]`, `\n` and `**` are meant to follow the [guidelines of the deployment pipeline](https://github.com/coveo/deployment-pipeline/blob/master/deployment_package/certifiers/package_rollout/audit_scm_changeset.py#L88).

Reply:
I see, thanks. It would be useful to have these links handy in the file, but it is a JSON file, hmm.

Based on the second link, should the `v` and `b` be capitalized in `Version Bump`?

Reply:
Lowercase works too, but I'll change it to be capitalized.